probe-rs/src/vendor/st/sequences/stm32_armv7.rs

```diff
@@ -54,6 +58,10 @@ mod dbgmcu {
         /// The offset of the Control register in the DBGMCU block.
         const ADDRESS: u64 = 0x04;
 
+        pub fn from_saved(contents: u32) -> Self {
```

Comment:
This function is identical to the constructor of the newtype struct and is not required

Reply:
Removed.